docs/src/pages/en/styling.md

```diff
@@ -10,14 +14,14 @@ Our current styling architecture is still quite nascent and is very likely to un
 
 ### Elements
 
-- `<media-captions-button/>` ([docs](./media-captions-button.md))
-- `<media-fullscreen-button/>` ([docs](./media-fullscreen-button.md))
-- `<media-mute-button/>` ([docs](./media-mute-button.md))
-- `<media-pip-button/>` ([docs](./media-pip-button.md))
-- `<media-play-button/>` ([docs](./media-play-button.md))
-- `<media-playback-rate-button/>` ([docs](./media-playback-rate-button.md)) ([See notes below \*](#notes))
-- `<media-seek-backward-button/>` ([docs](./media-seek-backward-button.md))
-- `<media-seek-forward-button/>` ([docs](./media-seek-forward-button.md))
+- `<media-captions-button/>` ([docs](./media-captions-button))
```

Comment:
too bad that we can have this link continue working in github too. If you figure it out, that'll be nice, but I wouldn't spend more than like a minute on this.

Reply:
ah yes, that would be nicer. I'll check this out